python/lsst/pipe/analysis/coaddAnalysis.py

```diff
@@ -1024,6 +1067,33 @@ def calibrateCatalogs(self, catalog, wcs=None):
         calibrated = calibrateCoaddSourceCatalog(catalog, self.config.analysis.coaddZp)
         return calibrated
 
+    def makeBadArray(self, catalog):
```

Comment:
looks like this method is an exact duplication of the `makeBadArray` method for the previous task.  It might be better to have a single function that both of these things call, rather then individual methods that might easily diverge from each other.